pkg/serverutils/utils.go

```diff
@@ -25,6 +26,28 @@ func SendResponse(rw http.ResponseWriter, code int, resp interface{}) {
 	}
 }
 
+func CheckUnsupportedBrowser(r *http.Request) bool {
```

Comment:
I would prefix it with `Is` to make it clear it's a function that returns a boolean.

```suggestion
func IsUnsupportedBrowser(r *http.Request) bool {
```